zstacklib/zstacklib/utils/upload_task.py

```
@@ -0,0 +1,330 @@
+def _fail_task(task, reason):
+    task.fail(reason)
+    raise Exception(reason)
+
+def upload_slice(entity, param, task):
+    # type: (Entity, UploadParam, UploadTask) -> None
+
+    boundary = get_boundary(entity)
+    if not boundary:
+        _fail_task(task, 'unexpected post form')
+
+    pool, _ = task.tmpPath.split('/')
```

Ruff (0.13.1)

Unpacked variable `pool` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)